.github/workflows/tests.yml

```diff
@@ -4,13 +4,17 @@ on: [push]
 jobs:
     cs:
       runs-on: ubuntu-latest
+      strategy:
+          matrix:
+              php: ['8.1', '8.2']
```

Comment:
I don't think it's necessary to execute the coding style control with multiple PHP versions.